.ruby-version

```diff
@@ -1 +1 @@
-2.2.3
+2.3.0
```

Comment:
does this mean that JT _requires_ ruby 2.3.0 now? if so that's changelog worthy

Reply:
nope. I don't even think it makes sense for gems to have this file. The required ruby dependencies are specified in the gemspec. This file is only used for codeship lol.